components/gorgias/gorgias.app.mjs

```diff
@@ -107,27 +146,67 @@ export default {
         cursor = meta.next_cursor;
       } while (cursor);
     },
-    async getEvents(params) {
+    async getEvents({
+      $, params,
+    }) {
       return this._makeRequest({
+        $,
         path: "/events",
         params,
       });
     },
-    async listCustomers(params) {
+    async createCustomer({
+      $, data,
+    }) {
       return this._makeRequest({
+        $,
+        path: "/customers",
```

Comment:
Since there's already a forward slash inserted before the `path`, you can remove it here and in all other methods with `path`.
``url: `https://${this.$auth.domain}/api/${path}`,``
```suggestion
        path: "customers",
```